USA/CA/beaumont/DEPRECATD_beaumont_stat_scraper.py

```diff
@@ -0,0 +1,133 @@
+import requests
```

Comment:
Why is this file deprecated? If it's not needed anymore, should we just remove it?

Reply:
Removed